backend/stewards/serializers.py

```
@@ -2,6 +2,35 @@
 
+def feature_candidate_user_data(user, include_id=False):
+    from social_connections.serializers import (
+        PublicDiscordConnectionSerializer,
+        PublicGitHubConnectionSerializer,
+        PublicTwitterConnectionSerializer,
+    )
+
+    def _get(related_name, serializer_class):
+        try:
+            connection = getattr(user, related_name)
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)